nerfstudio/fields/nerfacto_field.py

```diff
@@ -124,24 +124,27 @@ def __init__(
             in_dim=3, num_frequencies=2, min_freq_exp=0, max_freq_exp=2 - 1, implementation=implementation
         )
 
-        encoder = HashEncoding(
-            num_levels=num_levels,
-            min_res=base_res,
-            max_res=max_res,
-            log2_hashmap_size=log2_hashmap_size,
-            features_per_level=features_per_level,
+        # if implementation--torch, then use torch.nn.Sequential
+        # if implementation==tcnn, then use efficient encoding+MLP combo
+        self.mlp_base = EncoderAndMLP(
+            in_dim=3,
+            encoder_type=HashEncoding,
+            encoder_params={
+                "num_levels": num_levels,
+                "min_res": base_res,
+                "max_res": max_res,
+                "log2_hashmap_size": log2_hashmap_size,
+                "features_per_level": features_per_level,
+            },
+            mlp_params={
+                "num_layers": num_layers,
+                "layer_width": hidden_dim,
+                "out_dim": 1 + self.geo_feat_dim,
+                "activation": nn.ReLU(),
```

Comment:
it seems like this should be passing nn.ReLU instead of an instance, is there a reason it needs to call the constructor here?